Assets/Scripts/BuffPanelSDK/BuffPanel.cs

```diff
@@ -11,8 +11,9 @@ class BuffPanelMonoBehaviour : MonoBehaviour
 
     public class BuffPanel
     {
-        public static string serviceHostname = "buffpanel.com";
+        public static string serviceHostname = "staging.buffpanel.com:8080";
         public static string servicePath = "/api/run";
+        public static string redirectURI = ".redirect.staging";
```

Comment:
Be also advised that this needs to work correctly for aliases as well (i.e. like the news.spaceengineers.com and news.medievalengineers.com aliases we already use). Since the DNS CNAME record does not cause a redirect, the cookies will likely be stored under the alias domain.

Our current client has already requested an alias for her click links so this is an important concern. There are 3 solutions that come to mind:
1. We hardcode the aliases for the current client as a temporary measure and check for both the default and her custom alias.
2. We allow the client to specify any aliases upon the initialization of the SDK, and they will be checked together with our default.
3. We design a simple protocol, where the SDK uses the game token (which the client already provides) to request all the current aliases for the given game from the server, before it searches for the cookies.

I prefer the 3rd solution since it is the most future proof and simple for use when as we are planing to allow the user manage his aliases from our website (this would make them not need to release a new build once that happens).

For now, however, I think we have enough time only for the 1st or the 2nd, I leave it for you to decide which you'd rather implement.